README.md

```diff
@@ -1,2 +1,54 @@
-# Cellmata [![CircleCI](https://circleci.com/gh/d409f19/cellmata.svg?style=svg)](https://circleci.com/gh/d409f19/cellmata)
-This repo contains all relevant source code and files pertaining to the programming language Cellmata, developed by group d409f19.
+# Cellmata 
+![CircleCI branch](https://img.shields.io/circleci/project/github/d409f19/cellmata/master.svg)
+![GitHub release](https://img.shields.io/github/release/d409f19/cellmata.svg)
+![GitHub](https://img.shields.io/github/license/d409f19/cellmata.svg)
+
+Cellmata is a simple programming language designed for concisely writing and rendering cellular automata. 
+This project is developed on the basis of a semester-project at AAU by group d409f19. 
+
+This repository contains the source code and files related to the Cellmata compiler. 
+Note that all source-code is released under GNU GPL v2.0.
```

Comment:
I don't think it's normal to put the license it the opening block, I would expect it to be in at the end of the README under a license section

Reply:
Fixed in 69bfcf5